pkg/audit/service.go

```diff
@@ -30,6 +30,11 @@ type Config struct {
 	MinBytesPerSecond  memory.Size   `help:"the minimum acceptable bytes that storage nodes can transfer per second to the satellite" default:"128B"`
 	MinDownloadTimeout time.Duration `help:"the minimum duration for downloading a share from storage nodes before timing out" default:"5s"`
 	MaxReverifyCount   int           `help:"limit above which we consider an audit is failed" default:"3"`
+
+	ReputationAlpha0 float64 `help:"the initial shape 'alpha' used to calculate audit SNs reputation" default:"1.0"`
```

Comment:
why Alpha0 and Beta0 instead of just Alpha and Beta?

Reply:
The zero is meant to show that this is the initial value, rather than the value at the Nth measurement.